### PR TITLE
Tutorial scene with new tablet

### PR DESCRIPTION
**Pull request by ingolvjr**
got merge conflicts and trying to fix it by moving my changes to a separate scene

### CODE REVIEW

**Comment by ingolvjr** [comment]
Fixed conflicts in f5bc57fbf9368ee4451f82c30cb376100cb493f0

File: Assets/VR4VET/Components/Tutorial/Scripts/ControllerTooltips.cs
Comment (SnorreForbregd) [inline Assets/VR4VET/Components/Tutorial/Scripts/ControllerTooltips.cs:1]: ControllerTooltips.cs should not be deleted

File: Assets/VR4VET/Components/Tutorial/Scripts/ControllerTooltips.cs.meta
Comment (SnorreForbregd) [inline Assets/VR4VET/Components/Tutorial/Scripts/ControllerTooltips.cs.meta:1]: ControllerTooltips.cs should not be deleted, and unity needs the .meta files

**Comment by fominykh** [comment]
What is the status @SnorreForbregd ? Does it make sense to wait for the next release of the tablet before addressing this issue?

**Pull request closed without merging**